src/app/component/layout/components/footer/footer.component.scss

```diff
@@ -56,7 +56,12 @@ li {
 .footer_social-link {
   width: 16px;
   height: 16px;
-  margin: 0 7px;
+  margin: 0 6px;
+
+  img {
+    width: 15px;
+    height: 12.19px;
+  }
 }
```

Comment:
Do you realy need this 0.19px after 12 ?

Reply:
Yea, it's not my whim))). It was like a restriction, so yeah, we really need it :D